### PR TITLE
[TM] Azure component and API parameter change.

### PR DESCRIPTION
**Pull request by adamhuang97**
Change(s):
   - Change oat risk level.
   - Fix template for application.
   - Fix log wording.

   Reason for Change(s):
   - Azure application insight change workspace base.
   -  Change log wording.

   Version Updated:
   - No

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by vmanojreddy** [comment]
@adamhuang97  Tested the data connector by deploying the function app. we see error for function. Please check and update. thanks!!!
image

**Comment by vmanojreddy** [comment]
@adamhuang97  Tested the data connector by deploying the function app. we see error for function. Please check and update. thanks!!!
image

**Comment by adamhuang97** [comment]
Hi @vmanojreddy,
It seems auth failed with api token auth with API,
Since it need production key, may I provide you in other mail or private thread?

Thank you.
Best regards,
Adam

**Comment by adamhuang97** [comment]
Hi @vmanojreddy 
May I know is there any update?
Thank you.
Best regards,
Adam

**Comment by vmanojreddy** [comment]
@adamhuang97 Tested the data connector by deploying the function app again. we are getting same error for function. Could you please check it and update it. thanks!!!
image

**Comment by adamhuang97** [comment]
Hi @vmanojreddy 
Since the error cause by error auth with our production service.
It need an API token to connect with our server.
May I provide you the token with MAIL or private channel?

Thank you.
Best regards,
Adam

**Comment by adamhuang97** [comment]
Hi @vmanojreddy,
Is any update for this PR?
Since we could not provide api token in this public thread.
Thank you.
Best regards,
Adam

**Comment by NikTripathi** [comment]
Hi @adamhuang97 There are conflicts in this branch can you please resolve?

**Comment by NikTripathi** [comment]
Hi @adamhuang97 ,
Functions are not loading when we target website run from package link to the zip package added in this PR. 
image

**Comment by adamhuang97** [comment]
Hi @NikTripathi ,
I don't have permission to resolve conflict on this portal...

May I know what means "target website run from package link to the zip package added in this PR."?
Because last reviewer could verify when function triggered.

Thank you.
Best regards,
Adam

**Comment by NikTripathi** [comment]
> Hi @NikTripathi , I don't have permission to resolve conflict on this portal... Screen Shot 2022-05-20 at 8 20 03 PM
> 
> May I know what means "target website run from package link to the zip package added in this PR."? Because last reviewer could verify with function triggered.
> 
> Thank you. Best regards, Adam

Hi @adamhuang97 
image
In this we use the url which targets to the zip package. When I am using the url for the zip package you have added in the PR function are not loading.

**Comment by adamhuang97** [comment]
Hi @NikTripathi 
Since data connector move to solution path.
I've resolved conflict, would you please help to try again?

Thank you.
Best regards,
Adam

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -1211,12 +1211,12 @@
...
     "workbookKey": "TrendMicroXDR",
     "logoFileName": "trendmicro_logo.svg",
-    "description": "Gain insights from Trend Micro XDR with this overview of the Alerts triggered.",
+    "description": "Gain insights from Trend Micro Vision One with this overview of the Alerts triggered.",
     "dataTypesDependencies": [ "TrendMicro_XDR_WORKBENCH_CL" ],
     "dataConnectorsDependencies": [ "TrendMicroXDR" ],
     "previewImagesFileNames": [ "TrendMicroXDROverviewWhite.png", "TrendMicroXDROverviewBlack.png" ],
-    "version": "1.2.0",
-    "title": "Trend Micro XDR Alert Overview",
+    "version": "1.3",
```
Comment (NikTripathi) [inline Workbooks/WorkbooksMetadata.json:1218]: Please use a three digit version.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

File: Workbooks/WorkbooksMetadata.json (hunk 1 above)
Comment (adamhuang97) [reply Workbooks/WorkbooksMetadata.json]: Hi @NikTripathi 
Fixed.
Thank you.

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**